src/main/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTO.java

```diff
@@ -1,31 +1,28 @@
 package org.dcsa.bkg.model.transferobjects;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
 import org.dcsa.core.events.model.enums.PartyFunction;
 import org.dcsa.core.events.model.transferobjects.PartyTO;
 import org.dcsa.core.exception.InvalidParameterException;
 
-import javax.validation.constraints.NotEmpty;
+import javax.validation.Valid;
 import javax.validation.constraints.NotNull;
 import java.util.List;
 
 @Data
 public class DocumentPartyTO {
 
+  @Valid
+  @NotNull(message = "Party is required.")
   private PartyTO party;
 
   @NotNull(message = "PartyFunction is required.")
   private PartyFunction partyFunction;
 
   private List<String> displayedAddress;
 
-  @NotNull(message = "PartyContactDetails is required.")
-  @NotEmpty(message = "PartyContactDetails is required.")
-  private List<PartyContactDetailsTO> partyContactDetails;
```

Comment:
why is this removed? The OAS still has PartyContactDetails

Reply:
It's moved to PartyTO as it was more relevant to a party object as discussed with Henrik.